src/TriggerBinding/SqlTriggerListener.cs

```diff
@@ -175,6 +173,9 @@ public async Task StartAsync(CancellationToken cancellationToken)
                     };
 
                     TelemetryInstance.TrackEvent(TelemetryEventName.StartListenerEnd, this._telemetryProps, measures);
+
+                    this._scaleMonitor = new SqlTriggerScaleMonitor(this._userFunctionId, this._userTable.FullName, this._connectionString, this._maxChangesPerWorker, this._logger);
```

Comment:
I see that the default MaxChangesPerWorker, which is used as the target value, is 1000. This is quite high (defaults for Service Bus and Event Hubs are 16 and 10 respectively). 

What range of values would you expect UnprocessedChangeCount for an average customer to be? If this is well under 1000 I would consider lowering the default value.

Reply:
This is something we'll be monitoring with our telemetry and will adjust as needed, for now though 1000 as the default doesn't seem unreasonable as 1000 rows is generally a very manageable amount of data for a single worker to handle.

Do you have information on how Service Bus and Event Hubs arrived at those values for their use cases? Is there some expectation that each change will add enough extra overhead that extra workers need to be spun up?